features/dd-sdk-android-flags-openfeature/build.gradle.kts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import com.datadog.gradle.config.androidLibraryConfig
+import com.datadog.gradle.config.dependencyUpdateConfig
+import com.datadog.gradle.config.detektCustomConfig
+import com.datadog.gradle.config.javadocConfig
+import com.datadog.gradle.config.junitConfig
+import com.datadog.gradle.config.kotlinConfig
+import com.datadog.gradle.config.publishingConfig
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
+plugins {
+    // Build
+    id("com.android.library")
+    kotlin("android")
+    id("com.google.devtools.ksp")
```

Comment:
are we going to generate anything in this module based on `@NoOpImplementation` annotation?

Reply:
no.